src/modules/file/file.repository.ts

```diff
@@ -43,6 +43,10 @@ export class FileModel extends Model implements FileAttributes {
   @Column
   name: string;
 
+  @Index
+  @Column
+  plain_name: string;
```

Comment:
camelCase here too, as we use `underscored: true`, which handles the conversion from camelCase to snake_case for us